Code/Tools/ProjectManager/Platform/Mac/PAL_mac.cmake

```
@@ -5,3 +5,15 @@
 # SPDX-License-Identifier: Apache-2.0 OR MIT
 #
 #
+
+add_executable(O3DE_SDK Platform/Mac/O3DE_SDK_Launcher.cpp)
+
+set_target_properties(O3DE_SDK PROPERTIES
+    XCODE_ATTRIBUTE_ENABLE_HARDENED_RUNTIME YES
+    XCODE_ATTRIBUTE_CODE_SIGN_INJECT_BASE_ENTITLEMENTS NO
+)
+
+ly_target_link_libraries(O3DE_SDK
+    PRIVATE
+        AZ::AzCore
+        AZ::AzFramework)
```

nitpick: the ending parenthesis should be on a separate line and the file should end in a newline
```suggestion
        AZ::AzFramework
)

```

---

Fixed